packages/amplify-graphql-transformer-interfaces/package.json

```diff
@@ -48,6 +48,7 @@
       "node"
     ],
     "collectCoverage": true,
+    "coverageProvider": "v8",
     "coverageReporters": [
       "clover",
       "text"
```

Comment:
Understanding that this is an interface package and has no executable code, should we defensively code a threshold limit here to catch any future changes, or is there a global setting somewhere that applies in the absence of a specific configuration for this package?

Reply:
No, we do not have any global threshold. Since most of these types (or constants) are used in other projects, we do not have any threshold check for this package.